addons/rose/addon/components/rose/form/actions/edit-toggle/index.hbs

```diff
@@ -3,8 +3,11 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<div class='rose-form-actions' ...attributes>
-  <Rose::Button @style='secondary' {{on 'click' @enableEdit}}>
-    {{@enableEditText}}
-  </Rose::Button>
+<div class='rose-form-actions'>
```

Comment:
I kept this class name for now because a lot of acceptance tests use this in the selector. Thoughts on later removing this class name, keeping it, or using an ember data-test selector instead (for a future refactor)?

Reply:
We should still probably include `...attributes`.

Reply:
What's the context behind the idea to change the class name?

Reply:
Right now the only places the class name is being used is in acceptance tests so that's why I wanted to get thoughts on later refactoring to just use a data test selector. However, a point for keeping it, is to facilitate adding custom styles in the future.

Reply:
thanks for clarifying, I vote for data selectors and getting rid of the class name, since I do not anticipate custom styles to form action buttons.

Reply:
Nice clarification! I am 100% on @DhariniJeeva point, +1 to data selectors :wink: